src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,6 +5,14 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private String engineType;
+    private String model;
```

Comment:
Don't forget about access modifiers for your custom fields in `Excavator`. It's good practice to use private access modifiers and provide getters and setters for accessing these fields.